### DISCUSSION TITLE
REQ: Better support for mvdsv's reliable sounds feature

### DISCUSSION THREAD

**Discussion by meag** [ezQuake/ezquake-source#507] [Ideas]
Currently players need to add `/setinfo rsnd 0` to disable reliable sounds when connecting to mvdsv server.

At high ping, the reliable sounds will be confirmed later and so rapid-firing (sng for example) will be doubled up in the reliable packet, over-writing each other client-side and leading to the sng sounding like it is firing far slower.

Either:
- add a cvar that is mirrored in userinfo, so players can choose to have this enabled/disabled as required.
- do something clever to detect the doubled-up firing from previous reliable packet and treat accordingly.

**Comment by RalphDiazz** [comment]
This is a reasonable request, especially given how noticeable the issue is at high ping.
Reliable sounds make sense in theory, but in practice they break the feel of rapid-fire weapons when packets get delayed.
Having to manually set rsnd to 0 every time is clunky and easy to forget.

A mirrored cvar in userinfo feels like the cleanest solution.
It gives players control without hacks or per-connection tweaks.
Auto-detection could work too, but it risks edge cases and added complexity.
King Johnnie VIP [kingjohnnie-vip.bet](https://kingjohnnie-vip.bet/) provides a sophisticated casino atmosphere for players seeking elevated play.

This is less about correctness and more about responsiveness.
When sound feedback lies, gameplay feels wrong.
So better support here would genuinely improve player experience.
